### ISSUE TITLE
Text properties for track layer

### ISSUE THREAD

**Issue by neili02** [napari/napari#4066]
It does not appear that there is currently an easy way to change the properties of the track id label text that is displayed when `display_id` is set to `True`. Looks like the font settings are hard coded in the TracksVisual class. Would be nice for the track layer to have a `text` parameter similar to the Points layer that allows for specifying the properties of the track id text label.

Thanks so much for creating and maintaining this awesome project.

**Comment by jni** [comment]
Yup, I think this was implicitly already on @andy-sweet's "features" radar, but thank you for raising it @neili02! We will definitely implement this as soon as we can. See #2866 for links to many related issues.
